content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -54,6 +54,16 @@ The upgrade workflow at high level is the following:
   with the purpose of reconfiguring the cluster is not recommended and can have unexpected results. Follow the steps in
   [Reconfiguring a kubeadm cluster](/docs/tasks/administer-cluster/kubeadm/kubeadm-reconfigure) instead.
 
+### Considerations when upgrading etcd
+
+Because the `kube-apiserver` pod is running at all times (even if you have
+drained the node), when you perform a kubeadm upgrade which includes an etcd
+upgrade, in-flight requests will stall while the new etcd pod is restarting.
+As a workaround, it is possible to actively stop (with `kill -s SIGTERM`)
```

Comment:
maybe we can have a more detailed example.

----------

text, text, text use these commands:
```
command 1
command 2
```
more text

Reply:
good point. done